frontend/template-partials/views/partials/maincontent-left.html

```diff
@@ -1,10 +1,10 @@
 {{$content-outer}}
 {{$preloader}}{{/preloader}}
-  <div class="column-two-thirds">
+  <div class="govuk-grid-column-two-thirds">
     {{$validationSummary}}{{/validationSummary}}
-    <header>
-      <h1>{{$header}}{{/header}}</h1>
-    </header>
+    {{#subHeading}}<span class="govuk-caption-l">{{subHeading}}</span>{{/subHeading}}
+    {{#header1}}<h1 id="page-header" class="govuk-heading-l">{{header1}}</h1>{{/header1}}
+    {{#header}}<h1 class="govuk-heading-xl">{{header}}</h1>{{/header}}
```

Comment:
you don't really have multiple h1 tags like this.  It would be confusing to a screenreader user because you are jumping to the same section

Reply:
this has some issues.  According to html5 you should avoid using multiple h1 tags but if you do you wouldn't put it like the above.  I suggest making it a div.  Also it doesn't quite make sense from a front end to have the extra large heading after the large heading as per also gds's guidance https://design-system.service.gov.uk/styles/typography/.  Here is what I propose as a change
{{$content-outer}}
{{$preloader}}{{/preloader}}
  <div class="govuk-grid-column-two-thirds">
    {{$validationSummary}}{{/validationSummary}}
    {{#captionHeading}}<span class="govuk-caption-l">{{captionHeading}}</span>{{/captionHeading}}
    {{#header}}<h1 class="govuk-heading-xl">{{header}}</h1>{{/header}}
    {{#subHeading}}<div id="page-header" class="govuk-heading-l">{{subHeading}}</div>{{/subHeading}}
    {{$content}}{{/content}}
  </div>
{{/content-outer}}